akka-discovery/src/test/scala/akka/discovery/config/ConfigServiceDiscoverySpec.scala

```diff
@@ -38,7 +38,7 @@ akka {
             }
           ]
         },
-        service2 = {
+        "service2.domain.com" = {
```

Comment:
Simple and straightforward. 

I see that we were not using "service2" in the tests and I'm wondering why we have it in the config.

Reply:
Would be good with a test case actually using this as well (even if the bug was that it failed already while parsing?)

Reply:
I think it is since it makes it clear that both service blocks are there for the test, rather than implicitly knowing that all such blocks will be parsed eagerly.

Reply:
You are right, I will add a test that exercises that second config!